openstf/model/split_energy.py

```diff
@@ -42,23 +44,65 @@ def split_energy(pid):
     error = components[["load", "Inschatting"]].diff(axis=1).iloc[:, 1]
     mae = error.abs().mean()
     coefdict.update({"MAE": mae})
+    coefsdf = convert_coefdict_to_coefsdf(pj, input_split_function, coefdict)
 
-    # Get average coefs of previous runs
+    # Get average coefs of previous runs and check if new coefs are valid
     mean_coefs = db.get_energy_split_coefs(pj, mean=True)
```

Comment:
Do we still want the mean of the old coefs? Wouldn't just the most recent coefs make more sense?

Reply:
In addition, you wouldnt need L60-64

Reply:
1. I kept the mean instead only the last coefs, in case the last ones were odd. I don't have a specific preference.
2. Because I am using the mean and not the last ones, I compare the coefs in dict-format with `mean_coefs` in dict-format. After that, I do not use those values further and get the last coefs.

Looking at this answer, it does make more sense to use the last ones, I thought the mean would have a preference, but that does not seem to be the case. I will compare the coefficients with the last ones.